lib/Data/Content/ContentUpdateData.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/**
+ * This file is part of the eZ RepositoryForms package.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ * @version //autogentag//
+ */
+namespace EzSystems\RepositoryForms\Data\Content;
+
+use eZ\Publish\Core\Repository\Values\Content\ContentUpdateStruct;
+use EzSystems\RepositoryForms\Data\NewnessCheckable;
+
+/**
+ * @property-read \EzSystems\RepositoryForms\Data\Content\FieldData[] $fieldsData
+ * @property-read \eZ\Publish\API\Repository\Values\ContentType\ContentType $contentType
+ * @property-read \eZ\Publish\API\Repository\Values\Content\Content $contentDraft
+ */
+class ContentUpdateData extends ContentUpdateStruct implements NewnessCheckable
```

Comment:
It would be convenient, from the ContentEditView perspective, if both `ContentUpdateData` and `ContentCreateData` could implement a common interface (`ContentData` ?).

While at it, maybe `ContentData` could expose a `getContentType()` method. I could us that, and it would give this sucker something to do instead of just being there eating memory ;)